Tools/CISupport/ews-build/steps.py

```diff
@@ -2381,7 +2381,7 @@ class RunResultsdbpyTests(shell.ShellCommandNewStyle):
     ]
 
     def __init__(self, **kwargs):
-        super(RunResultsdbpyTests, self).__init__(timeout=2 * 60, logEnviron=False, **kwargs)
+        super(RunResultsdbpyTests, self).__init__(timeout=15 * 60, logEnviron=False, **kwargs)
```

Comment:
unit-test needs to be updated , see https://ews-build.webkit.org/#/builders/20/builds/102444